pkg/driver/s3-driver.go

```diff
@@ -136,7 +137,7 @@ func newNodeServer(d *S3Driver, statsUtil pkgUtils.StatsUtils, nodeID string, mo
 	}
 }
 
-func (driver *S3Driver) NewS3CosDriver(nodeID string, endpoint string, s3cosSession s3client.ObjectStorageSessionFactory, mountObj mounter.NewMounterFactory, statsUtil pkgUtils.StatsUtils) (*S3Driver, error) {
+func (driver *S3Driver) NewS3CosDriver(nodeID string, endpoint string, s3cosSession s3client.ObjectStorageSessionFactory, mountObj mounter.NewMounterFactory, statsUtil pkgUtils.StatsUtils, mounterUtil mounterUtils.MounterUtils) (*S3Driver, error) {
```

Comment:
You need to pass `mounterUtil mounterUtils.MounterUtils` in `newNodeServer()` call and set the value of `MounterUtils` there in struct 

Without this change, it will panic in nodserver while calling FuseUnmount method